app/src/main/java/org/breezyweather/common/source/LocationPreset.kt

```diff
@@ -60,6 +60,7 @@ enum class LocationPreset(
     ITALY("meteoam", airQuality = "openmeteo", pollen = "openmeteo", minutely = "openmeteo", normals = "accu"),
     NORWAY("metno", pollen = "openmeteo", alert = "accu", normals = "accu"),
     SWEDEN("smhi", airQuality = "openmeteo", pollen = "openmeteo", minutely = "metno", alert = "accu", normals = "accu"),
+    UNITED_KINGDOM("metoffice", current = "openmeteo", airQuality = "openmeteo", pollen = "openmeteo", minutely = "openmeteo", alert = "accu", normals = "accu"),
```

Comment:
I don't think we should recommand MET Office as the default source. Terms and conditions say:

> You may make no more than 5000 data requests per day; and
> You may make no more than 100 data requests per minute.

Which seem to be a limit Breezy Weather users will quickly reach, prompting all users at the same time to register an API key, which most users don't want to do. Our current recommandation for UK is Open-Meteo which has no enforced rate limit atm and already uses UK MET Office data, in combination with other models when data is missing, so it should be equally good, if not better ;)

Reply:
I just registered and it's even worse than I thought: it's 360 calls / day and 10K calls / months. Each user will have to register its own key (or just use Open-Meteo).